Networking/Networking/Remote/MediaRemote.swift

```diff
@@ -40,19 +40,25 @@ public class MediaRemote: Remote {
     ///
     /// - Parameters:
     ///     - siteID: Site for which we'll upload the media to.
+    ///     - productID: Product for which the media items are first added to.
     ///     - context: Display or edit. Scope under which the request is made;
     ///                determines fields present in response. Default is Display.
     ///     - mediaItems: An array of uploadable media items.
     ///     - completion: Closure to be executed upon completion.
     ///
     public func uploadMedia(for siteID: Int64,
+                            productID: Int64,
                             context: String? = Default.context,
                             mediaItems: [UploadableMedia],
                             completion: @escaping ([Media]?, Error?) -> Void) {
         let parameters = [
             ParameterKey.contextKey: context ?? Default.context,
         ]
 
+        let formParameters: [String: String] = [Int](0..<mediaItems.count).reduce(into: [:]) { (parentIDsByKey, index) in
+            parentIDsByKey["attrs[\(index)][parent_id]"] = "\(productID)"
```

Comment:
Does this type of key follow some API specification, or is it a multipart standard? It can be useful for the future adding a comment on how it's working (also a link to the doc it's ok). 😃

Reply:
yes! this is following the [API doc](https://developer.wordpress.com/docs/api/1.1/post/sites/%24site/media/new/ ) that is also linked in the function-level comment:

https://github.com/woocommerce/woocommerce-ios/blob/d7328be8c0cc07904f0f2a066ceb3976157bc46a/Networking/Networking/Remote/MediaRemote.swift#L38-L48